raster-foundry/raster-foundry
# Rewrite leaflet side-by-side

## Overview
Leaflet side-by-side does not support IE, and we want to support more complex interactions than the plugin is capable of.  This version should act identically to the original with respect to current functionality, with a design that will allow us to support more complex interactions such as horizontal splits down the road
Other changes:
* Remove Leaflet Side-By-Side & related styles

### Checklist

~- [ ] Styleguide updated, if necessary~
~- [ ] Swagger specification updated, if necessary~
~- [ ] Symlinks from new migrations present or corrected for any new migrations~

### Demo
![image](https://user-images.githubusercontent.com/4392704/28725790-8e0e0b6a-738c-11e7-96da-5758c6d6fbe7.png)

### Notes

The update cycle has not been fully optimized yet, so some events are probably being executed. I haven't noticed any issues in responsiveness though, even using Edge inside a vm, and I dedupe events before passing them onto angular, so it probably only a few extra cycles. 

## Testing Instructions

 * Verify that the split map functions identically to before. see #2293

Closes #2276
